.github/workflows/publish_collection.yml

```diff
@@ -0,0 +1,58 @@
+name: Publish fedora.linux_system_roles collection to Ansible Galaxy
+
+on:
+  schedule:
+    # Run daily at 00:00 UTC
+    - cron: '0 0 * * *'
+
+jobs:
+  publish_collection:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout auto-maintenance repository
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+
+      - name: Install dependencies
+        shell: bash
+        run: |
+          set -ex
+          pip install --upgrade pip
+          pip install --upgrade ansible ansible-galaxy galaxy-importer
+          docker --version
+
+      - name: Setup paths
+        id: locations
+        shell: bash
+        run: |
+          echo "dest_path=/var/tmp/collection" >> $GITHUB_OUTPUT
+
+      - name: Build and publish the collection
+        shell: bash
+        run: |
+          set -exu
+          # Ensure there is no dest_path before running release_collection.py
+          rm -rf ${{ steps.locations.dest_path }}
+          python ./release_collection.py --debug --dest-path ${{ steps.locations.dest_path }}
+          # We are up to date - exit
+          if git diff --quiet; then
+            echo "No roles have new releases - no collection will be published"
+            exit 0
+          fi
+          # A new collection has been build - find the tarball
+          _tarballs=( $(find ${{ steps.locations.dest_path }} -maxdepth 1 -type f -name '*.tar.gz') )
+          if [[ ${#_tarballs[@]} -eq 0 ]]; then
```

Comment:
I might be worrying too much, but it makes me feel better if you exit with 1 when you find 2 or more tarballs. :) Thanks!